src/pages/NotFoundPage.vue

```diff
@@ -12,15 +12,15 @@
 import HeaderSection from '@/components/HeaderSection';
 import FooterSection from '@/components/FooterSection';
 
-const HomePage = {
```

Comment:
Why are you renaming the HomePage component? There is already a NotFoundPage component, and even if there wasn't, you would just create tests for the HomePage component instead of renaming HomePage and writing tests for the renamed component :/

Reply:
I might have gotten confused because there's an already existing homepage component in `src/pages`, I'll do the necessary changes and make a new PR. Thanks for pointing it out.